lib/node_modules/@stdlib/fs/resolve-parent-paths-by/benchmark/benchmark.js

```diff
@@ -0,0 +1,349 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var basename = require( 'path' ).basename;
+var bench = require( '@stdlib/bench' );
+var pkg = require( './../package.json' ).name;
+var resolveParentPathsBy = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg+':mode=first', function benchmark( b ) {
+	var PATHS;
+	var opts;
+	var i;
+
+	PATHS = [
+		[ 'package.json' ],
+		[ 'beep-boop!!!hello world!?!', 'package.json' ],
+		[ 'package.json', 'README.md' ]
+	];
+	opts = {
+		'dir': __dirname,
+		'mode': 'first'
+	};
+
+	i = 0;
+	b.tic();
+
+	return next();
+
+	function next() {
+		i += 1;
+		if ( i <= b.iterations ) {
+			return resolveParentPathsBy(PATHS[ i % 3 ], opts, predicate, done);
```

Comment:
```suggestion
			return resolveParentPathsBy( PATHS[ i % 3 ], opts, predicate, done );
```

update the rest of the file accordingly